imports/wasi_snapshot_preview1/poll.go

```diff
@@ -60,6 +68,11 @@ func pollOneoffFn(ctx context.Context, mod api.Module, params []uint64) syscall.
 		return syscall.EFAULT
 	}
 	outBuf, ok := mem.Read(out, nsubscriptions*32)
+	// zero-out all buffer before writing
+	for i := range outBuf {
```

Comment:
I checked and seems this is idiomatic for filling with zeros https://groups.google.com/g/golang-nuts/c/8WSkwEs4YkU

Reply:
you mean `copy(arr, make([]int, len(arr)))` right?

Reply:
I mean what you are doing seems better than that, yeah (as it avoids alloc and I don't see anyone saying to do anything otherwise really)

Reply:
oh ok 👍